src/lib/api/coaching-relationships.ts

```diff
@@ -1 +1,54 @@
 // Interacts with the coaching_relationship endpoints
+
+import { CoachingRelationshipWithUserNames,
+            coachingRelationshipsWithUserNamesToString,
+            defaultCoachingRelationshipsWithUserNames,
+            isCoachingRelationshipWithUserNamesArray
+        } from "@/types/coaching_relationship_with_user_names";
+import { Id } from "@/types/general";
+import { AxiosError, AxiosResponse } from "axios";
+
+export const fetchCoachingRelationshipsWithUserNames = async (
+    organizationId: Id
+  ): Promise<[CoachingRelationshipWithUserNames[], string]> => {
+    const axios = require("axios");
+  
+    var relationships: CoachingRelationshipWithUserNames[] = defaultCoachingRelationshipsWithUserNames();
```

Comment:
Nice use of function-scoping here. :) I'd think your Rust training made the `let` keyword your default.

Reply:
Why thank you. I'm influenced by Rust quite a bit. :)